golem-test-framework/src/config/cli.rs

```diff
@@ -425,6 +430,10 @@ impl CliTestDependencies {
                 .await,
             );
 
+
+        let blob_storage = Arc::new(FileSystemBlobStorage::new(&PathBuf::from("/tmp/ittest-local-object-store/golem")).await.unwrap());
```

Comment:
This is consistent with what the other services use for their blob_storage, but this looks a bit broken to me. I think it would be better to have somethink like minio running and use the s3 based blob storage for all services

Reply:
The test framework is used for multiple things:
- Worker executor tests (`golem-worker-executor-tests/tests`) for testing various properties of the executor (only). For this minimal dependencies and as fast startup time as possible are the most important for me so here we use file system / in memory implementations
- Integration tests involving all services (`integration-tests/tests`). For these I agree that it would be better to use the s3 based blob storage. 
- CLI tests (`golem-cli/tests`) are like integration tests but doing operations through invoking the CLI itself. For historical reasons many things we should have as integration tests are implemented as CLI tests now - and for those I would also use S3 instead of file system storage, but if we think about it as just tests for the CLI layer itself it should not matter
- Benchmarks - we are running those either against locally a started golem cluster where I think using minio instead of the local file system would not be very useful (not interested in the performance of minio itself). And of course we plan to be able to run them against a real cluster deployed somewhere (but currently not doing it)

That said there are various ways to set up the test dependencies, as you see, and the `cli.rs` one is only used for benchmarks at the moment.

Reply:
What I mean with broken here is that we setup the blob storage as using a filesystem in the containers /tmp/ittest-local-object-store/golem, but I don't see any volume mounts that would allow sharing this directory between containers.